Cargo.lock

```diff
@@ -240,6 +240,7 @@ dependencies = [
  "net_gen 0.1.0",
  "polly 0.0.1",
  "rate_limiter 0.1.0",
+ "serde 1.0.105 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
nit: this should be in the `devices: balloon: expose balloon API` commit - that's where you take `serde` as a dependency in `devices`